x-pack/plugins/security/public/management/role_mappings/edit_role_mapping/rule_editor_panel/json_rule_editor.tsx

```diff
@@ -97,7 +97,7 @@ export const JSONRuleEditor = (props: Props) => {
       data-test-subj="roleMappingsJSONEditor"
     >
       <Fragment>
-        <CodeEditorField
+        <CodeEditorField // TODO theme provided in context in x-pack/plugins/security/public/management/role_mappings/role_mappings_management_app.tsx
```

Comment:
Found an issue here. When the space default is dark and the user setting is light, the JSON rule editor in Role Mappings is rendered dark.
<img width="1438" alt="Screenshot 2023-07-07 at 2 26 48 PM" src="https://github.com/elastic/kibana/assets/103939324/d59854e6-fd94-4594-8b6a-e266d29ea3e4">

Reply:
Nice find! We will address this as discussed on Slack